# Complementation of *entire-2*

## Purpose

To test if complementation by a functional atPIN1 gene (from the patPIN1::atPIN1::GFP) is capable of rescuing the *entire-2 *phenotype. 

## Experimental Setup

### Progenitor of the population

#### Date July 2010
In my earlier work I had some problems with viruses. In a population that was segregating for the e2 (1:2:1) and PIN1::GFP (1:2:1), many of them had virus infection.  I genotyped the healthy ones and found one individual that had the presence of GFP using mGFP4-F and mGFP4-R primers and also genotyped  as homozygous for the *entire-2* phenotype, using the TaaI(HpyCH4III ferments) (July 13, 2010 page 30 of original notebook enzyme test on page 26).

### Experiment: Population n = 168
Date: Summer 2011

The population should be all homozygous for *entire-2* and segregating for atPIN1::GFP presence (3:1,(PIN1::GFP/PIN1::GFP or PIN1/+):(+/+)).  

#### Genotyping

*entire-2*
This was verified by randomly genotyping by sequencing 20 individuals using PIN1-f and PIN1-r primers (sequencing 6/22 & 6/29, 2012), they all were homozygous for the *entire-2* snp. (Jeanice's notebook page 65).

pAtPIN1::AtPIN1::GFP

In order to test for AtPIN presence we genotyped for GFP using the mGFP4-F and mGFP4 primers (In Jeanice's notebook page 31-33).

### Phenotyping
Plants were grown in chamber till around a month old, then switched to core greenhouse.  Plants were around 3 months old when we phenotypes and harvested leaves. 

They were scored on presence or absence of spiral phyllotaxy and all mature leaves were photographed and the number of leaflets was scored. 

Leaf 4 and Leaf 5 were photographed.

Directions for Rachis meaurements:

1. Download ImageJ: http://rsbweb.nih.gov/ij/download.html

2. Open file in Image J

3. Set Scale
    a. Zoom in on ruler, click on line tool and make a line 1 cm long. 
    b. Analyze->Set Scale-> in known distance type "1"
    c. in unit of length type "cm"

2. Things to measure

1.) rachis- the axis that holds all leaflets including the petiole. 
    a. Use free hand line tool
    b. Trace with bamboo pad
    c. press "command + M" to measure, results should appear in results         window
    d. measure each leaf 4 and 5 separately 

2.) terminal leaf area

3.)how many leaflets( primary, seconday, intercalary)

## Results

We scored the progeny of these individuals and as expected, approximately 3/4 (72%) of the population showed genotypic presence of GFP. If pPIN1::PIN1::GFP is capable of rescue we expect a rescue of 3/4 of the population. Scoring for presence of spiral phyllotaxy, the chi squared (.72) was not significantly different from the expected 3/4 segregation ratios. The GFP presence also segregated with the spiral phenotype, with 88% of those individuals with GFP having a spiral phenotype. We also found leaf complexity to be significantly (p=.01) higher in individuals with GFP presence than without GFP presence. These results confirm that a function PIN1 is capable of rescuing both leaf complexity and spiral phyllotaxy in individuals homozygous for the *enitre-2* loci.

### Additional Notes
Of the 45 which did show PIN1::GFP presences, 22 did have a spiral phenotype, indicating that some *entire-2* plants do have a spiral phenotype.

**How many *entire-2* plants display a spiral phenotype?**

Of the 117 plants with PIN1::GFP presence, 14 plants did not have the spiral phenotype, indicating some of the plants were not rescued. 

PIN1::GFP presence could not rescue leaf shape phenotype fully.  

