# PIN1 Phylogeny 

The main goal of this tree is to find the origin of the duplication in the  SoPIN1 clade in tomato. 

## Methods

We used only sequences from species from fully sequenced genomes to ensure proper representation of the PIN gene family. All cDNA sequences were retrieved from either Pytozome (Goodstein et al. 2012), or through BLAST searches using genome databases available on the Sol Genomics Network (Fernandez-Pozo et al. 2015).  Gene orthology was further confirmed through comparisons with previous phylogenetic work (Bennett et al. 2014; O’Connor et al. 2014). AtPIN3 (AT1g70940) was used as the outgroup. All sequences were aligned using MUltiple Sequence Comparison by Log- Expectation (MUSCLE) alignment (Edgar 2004) on the EMBL-EBI bioinformatics web server (Li et al. 2015).  Sequences were further trimmed using TrimAL (version 1.2rev59) with gap threshold set to 90% and a specified conservation minimum of 60% positions from the original alignment. Using the servers from The CIPRES Science Gateway (version 3.3; (Miller, Pfeiffer, and Schwartz 2010). Maximum likelihood analysis was performed using RAxML-HPC2 (version 8.1.24; (Stamatakis 2014). Visualization and editing of trees was accomplished using FigTree (version 1.4.2; (Rambaut and Drummond 2009). 

## Species

1. Solanum lycopersicum (heinz)
2. Solanum pennellii
3. Solanum habrochaites
4. Solanum pimpinellifolium
5. Capsicum annuum (BLAST)
6. Nicotiana benthamiana (BLAST) : draft genome
7. Nicotiana tabacum (BLAST): draft genome *removed*
8. Solanum tuberosum (BLAST)
9. Mimulus guttatus
10. Arabidopsis thaliana 
11. Cucumis sativus 
12. Populus trichocarpa 
13. Medicago truncatula 
14. Citrus clementina
15. Capsella rubella 

## References

Bennett, Tom, Samuel F. Brockington, Carl Rothfels, Sean W. Graham, Dennis Stevenson, Toni Kutchan, Megan Rolf, et al. 2014. “Paralogous Radiations of PIN Proteins with Multiple Origins of Noncanonical PIN Structure.” Molecular Biology and Evolution 31 (8): 2042–60.

Edgar, Robert C. 2004. “MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput.” Nucleic Acids Research 32 (5): 1792–97.

Fernandez-Pozo, Noe, Naama Menda, Jeremy D. Edwards, Surya Saha, Isaak Y. Tecle, Susan R. Strickler, Aureliano Bombarely, et al. 2015. “The Sol Genomics Network (SGN)—from Genotype to Phenotype to Breeding.” Nucleic Acids Research 43 (D1): D1036–41.

Goodstein, David M., Shengqiang Shu, Russell Howson, Rochak Neupane, Richard D. Hayes, Joni Fazo, Therese Mitros, et al. 2012. “Phytozome: A Comparative Platform for Green Plant Genomics.” Nucleic Acids Research 40 (Database issue): D1178–86.

Li, Weizhong, Andrew Cowley, Mahmut Uludag, Tamer Gur, Hamish McWilliam, Silvano Squizzato, Young Mi Park, Nicola Buso, and Rodrigo Lopez. 2015. “The EMBL-EBI Bioinformatics Web and Programmatic Tools Framework.” Nucleic Acids Research 43 (W1): W580–84.

Miller, M. A., W. Pfeiffer, and T. Schwartz. 2010. “Creating the CIPRES Science Gateway for Inference of Large Phylogenetic Trees.” In Gateway Computing Environments Workshop 